### ISSUE TITLE
Text is bigger on Linux than on Windows

### ISSUE THREAD

**Issue by likeawindrammer** [AttorneyOnline/AO2-Client#96]
Even when using the exact same theme, the text on Linux will be bigger than on Windows, cutting short some of the labels.
Linux: https://pasteboard.co/IrG3cOR.png
Wine: https://pasteboard.co/IrG3s8V.png
Windows:https://cdn.discordapp.com/attachments/591189481667821582/605889984733511718/foto_chistosa.jpg
Edit: While testing some older versions of AO (2.4), it seems this has always been an issue on Linux.

**Comment by oldmud0** [comment]
This is not an AO issue, this is a Qt font rendering issue. There is not much I can do about it, other than setting a font size override for Linux.

**Comment by likeawindrammer** [comment]
It seems AO is not setting a font for many interface elements, leaving the choice of font and point size entirely to QT, which in turn choose whatever is the system's default. On Windows the default seem to be Arial 8, while on Ubuntu it's Ubuntu(the font) 11. I don't know Mac's default font, but I suppose it's also different to Windows, so Mac users might be having a similar issue.

If I install Arial and add:
```
QFont _font("Arial");
_font.setPointSize(8);
QApplication::setFont(_font);
```
To AOApplication::construct_lobby() I pretty much get the same "look" as the windows users: https://pasteboard.co/IrHJlTV.png

With this in mind I figure it might be possible to fix this by adding some open source font similar to Arial to the default base installation and using this font by default at 8 point size or so. This way every platform will get the same font and size.

**Comment by oldmud0** [comment]
Well, I guess I'm not only wrong, I'm spectacularly wrong.

**Comment by OmniTroid** [comment]
These issues are also present on MacOS builds

**Comment by stonedDiscord** [comment]
try installing segoe ui and check how it looks with that

**Comment by likeawindrammer** [comment]
Here's a zip with 22 screenshots doing multiple comparations: http://www.mediafire.com/file/f1wg9boa5c1i1c1/tests.zip/file

First of all: Windows doesn't use Arial by default for UI. It used Tahoma 8 previous to Vista, and SegoeUI 9 on vista and newer.
Second: For some reason AO doesn't use SegoeUI 9 on the courtroom even if running with windows 7 and using the basic theme. On windows AO seems to use Tahoma 8 always on the courtroom. And SegoeUI 9 on the lobby if running vista or newer.
Third: If I make AO use SegoeUI 9 on the courtroom, some strings are cut short.

So, long story short we need a font that is very similar to Tahoma AND open source so we can distribute it inside AO.

**Comment by stonedDiscord** [comment]
![grafik](https://user-images.githubusercontent.com/10584181/62835933-385bd180-bc5e-11e9-95bb-b356933c91e1.png)
text is smaller on mac os mojave

**Comment by oldmud0** [comment]
Oh...! So that's why 1.7.5 and AO2 look different in the lobby, despite everything being arranged the same.

Garuda looks like a good drop-in replacement for non-Windows platforms.

**Comment by likeawindrammer** [comment]
Microsoft itself made a font with an open font license to replace SegoeUI. It's called Selawik and it can be found here: https://github.com/microsoft/Selawik
Comparation: https://pasteboard.co/Isl5VLp.png

Also apparently Wine made a replacement for Tahoma, but for what I could see it was wider than the original. And I'm not sure about it's copyright. But it can be found here https://source.winehq.org/source/fonts/

**Comment by oldmud0** [comment]
Unfortunately, Selawik does not look ready for use. It is missing all of the other Unicode glyphs.

**Comment by likeawindrammer** [comment]
Maybe Ubuntu font? It's pretty similar to Tahoma and it has support for a lot of languages.
![tahoma ubuntu](https://user-images.githubusercontent.com/31085911/62918794-fd40c780-bd5d-11e9-8c42-940228c31183.png)
![tahoma ubuntu 2](https://user-images.githubusercontent.com/31085911/62918799-003bb800-bd5e-11e9-8417-30b4ba4134e7.png)
[comparations.zip](https://github.com/AttorneyOnline/AO2-Client/files/3495508/comparations.zip)

**Comment by oldmud0** [comment]
Fonts are now theme-configurable in 2.8.
